### DISCUSSION TITLE
SOC Overview Customization

### DISCUSSION THREAD

**Discussion by ThreeEyedRaven-99** [Security-Onion-Solutions/securityonion#7805] [Unsupported Versions] [answered]
Hello there,

I would like to customize SOC overview page. I understand I have to change motd by copying it to /opt/so/saltstack/local/salt/soc/files/soc/

The things I want to display on SOC overview are:
1. Connected and disconnected Wazuh Agents
2. Top 5 Rules triggered
3. Recent 5 Alerts Escalated
4. Top 5 data sources

If anyone can give an example would be appreciated.
Thanks in advance :)

Sorry for my bad English :)

**Comment by dougburks** [comment, marked as answer]
The SOC Overview page is just simple markdown and doesn't support visualizations. You might be able to create hyperlinks to your desired queries in Hunt.
